mock-api/vpc.ts

```diff
@@ -45,6 +45,98 @@ export const vpc2: Json<Vpc> = {
 
 export const vpcs: Json<Vpc[]> = [vpc, vpc2]
 
+export const routerRoutes: Json<Array<RouterRoute>> = [
+  {
+    id: '51e50342-790f-4efb-8518-10bf01279514',
+    name: 'default1',
+    description: "VPC Subnet route for 'default'",
+    time_created: '2024-07-11T17:46:21.161086Z',
+    time_modified: '2024-07-11T17:46:21.161086Z',
+    vpc_router_id: 'fc59fb4d-baad-44a8-b152-9a3c27ae8aa1',
```

Comment:
You could make this and spread it into each of these to take some of the noise out

```ts
const routeBase = {
  time_created: '2024-07-11T17:46:21.161086Z',
  time_modified: '2024-07-11T17:46:21.161086Z',
  vpc_router_id: vpcRouter.id,
  kind: 'default',
}
```